jx_sqlite/__init__.py

```diff
@@ -273,3 +273,11 @@ def copy_cols(cols, nest_to_alias):
         {"gte": [{"length": "nested_path"}, 1]}
     ]}
 )
+
+
+json_types = {
+    "TEXT": "string",
+    "INTEGER": "integer",
+    "REAL": "double",
+    "TINYINT": "boolean"
+}
```

Comment:
was this moved from elsewhere?  Does this exist somewhere else?  It looks familiar.